src/main/java/step3/Attempt.java

```diff
@@ -0,0 +1,29 @@
+package step3;
+
+public class Attempt {
+    private int number;
+
+    public Attempt(int number) {
+        if(number < 0) throw new IllegalArgumentException();
+        this.number = number;
+    }
+
+    void make() {
+        number--;
+    }
+
+    boolean isEnd() {
+        if(number == 0) {
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        return number == 0;
```
요렇게 표현하는 방법으로 전체적으로 리팩토링 해봅시다 :)

Reply:
0은 상수로 빼면 어떨까요? [참고링크](https://blog.daum.net/question0921/1103)